models/hh_cond_beta_gap_traub.h

```diff
@@ -151,11 +150,10 @@ References
 
 .. [1] Traub RD and Miles R (1991). Neuronal Networks of the Hippocampus.
        Cambridge University Press, Cambridge UK.
-.. [2] http://modeldb.yale.edu/83319
```

Comment:
This reference should rather be 

Brette, R., et al  (2007). Simulation of networks of spiking neurons: A review of tools and strategies. J Comput Neurosci, 23, 349–398.

and then 3 and 4 should stay.

Reply:
updated references